packages/server/src/supertokens.ts

```diff
@@ -224,7 +221,7 @@ export class SessionContextClass implements SessionContext {
   isAuthorized(input?: any) {
     if (!this.userId) return false
 
-    return config.isAuthorized(this.roles, input)
+    return config.isAuthorized(input)
```

Comment:
`isAuthorized` should now take `this.publicData` as first argument

Reply:
@Kosai106 **UPDATE:** canary branch already changed `config.isAuthorized` implementation so only minimal changes should be needed in this PR for that

Reply:
Excellent - I haven't had time to work on this but I'll be sure to rebase. 👍